advanced-achievements-plugin/src/main/java/com/hm/achievement/config/ConfigurationParser.java

```diff
@@ -199,26 +203,30 @@ private void parseAchievements() throws PluginLoadError {
 				}
 			}
 		}
-
-		int categories = NormalAchievements.values().length + MultipleAchievements.values().length + 1
-				- disabledCategories.size();
-		logger.info("Loaded " + achievementsAndDisplayNames.size() + " achievements in " + categories + " categories.");
-		if (disabledCategories.size() == 1) {
-			logger.info(disabledCategories.size() + " disabled category: " + disabledCategories.toString());
-		} else if (!disabledCategories.isEmpty()) {
-			logger.info(disabledCategories.size() + " disabled categories: " + disabledCategories.toString());
-		}
 	}
 
 	/**
 	 * Parses all achievements for a given category or category + subcategory. Populates the sortedThresholds map.
-	 * 
-	 * @param path
-	 * @throws PluginLoadError
+	 *
+	 * @param path category or category.subcategory
+	 * @throws PluginLoadError If an achievement fails to parse due to misconfiguration.
 	 */
 	private void parseAchievements(String path) throws PluginLoadError {
+		if (disabledCategories.contains(path)) {
```

Comment:
Is this check relevant here? I would be inclined to do all checks outside `parseAchievements` (this is already the case for Multiple categories). Here we will end up checking for paths that might contain subcategories as well, but `disabledCategories` was designed to only contain top-level categories. ^^

Reply:
I guess it made more sense when *Commands* category was being parsed with this method.
Now it does not reduce code so it can be moved back.

Moved back.